src/apps/tetris.js

```diff
@@ -0,0 +1,54 @@
+import Tetris from "killing-time-react-tetris";
+import { matchTriggerQueries } from "../lib/utilityApis";
+
+function Renderer() {
+  return (
+    <div>
+      <h1>Tetris</h1>
+      <Tetris>
+        {({ HeldPiece, Gameboard, PieceQueue, points, linesCleared }) => {
+          // Render it however you'd like
+          return (
+            <div>
+              <HeldPiece />
+              <div>
+                <p>Points: {points}</p>
+                <p>Lines Cleared: {linesCleared}</p>
+              </div>
+              <Gameboard />
+              <PieceQueue />
+            </div>
+          );
+        }}
+      </Tetris>
```

Comment:
@Nahdus What is your motivation behind using `killing-time-react-tetris` package? The code you have used here is from the documentation of `react-tetris` [here](https://www.npmjs.com/package/react-tetris#usage) and not for the package you have installed.